modules/ossm-kiali-viewing-metrics.adoc

```diff
@@ -0,0 +1,36 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/ossm-observability.adoc
+////
+
+[id="ossm-viewing-logs_{context}"]
+= Viewing logs in the Kiali console
+
+You can view logs for your workloads in the Kiali console.  The *Workload Detail* page includes a *Logs* tab which displays a unified logs view that displays both application and proxy logs.
+
+To change the logging level on the logs displayed in Kiali, you change the logging configuration for the workload or the proxy.
+
+//How often are logs updated?
```

Comment:
logs are pulled from k8s, so they are available as soon as they are available in k8s.  As for the Kiali logs tab, it has the standard refresh interval type dropdown as found in other parts of Kiali.  Kiali does not offer a log "streaming" view, they are pulled in batches.